recipes/unicorn-binance-local-depth-cache/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "unicorn-binance-local-depth-cache" %}
+{% set version = "0.7.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/unicorn-binance-local-depth-cache-{{ version }}.tar.gz
+  sha256: 2ee1e2c199ae6310816af8aaf7c9f1eda2d2120446462eacb4f9dc7a371feb43
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+    - requests
+    - unicorn-binance-rest-api
+    - unicorn-binance-websocket-api >=1.40.5
```

Comment:
As you appear to be involved upstream, I'd strong recommend avoiding importing in `setup.py` as it can cause a lot of issues. It seems like you're only using it for the version, in which case I'd personally recommend using [`setuptools-scm`](https://pypi.org/project/setuptools-scm/) instead. There are a bunch of other way of handling this, another reasonably one is to include the version in `setup.py`/`setup.cfg` and then use `importlib.metadata` to expose it programmatically (though you still have the duplication between git tags and the `setup.*`).

Reply:
Thanks i will study this and clear this in upcoming releases

Reply:
(Just to be clear, there is no requirements for this on the conda-forge side. It's just a personal recommendation.)

Reply:
thanks for the hint :)

i am not aware about the possible issues... must read them first. if it makes sense i will change this!

Reply:
Same solution can be used in the other PRs by the way.